README.md

```diff
@@ -526,11 +526,33 @@ make docs
 ```
 
 ## Releasing
-release to pypi by bumping the version to something reasonable and running
+You will need to have twine installed, and a `.pypirc` file in your home directory.
```

Comment:
How do we install twine?

Reply:
`pip3 install twine` -- I can update it to say "with pip" ?

Reply:
It would be nice to say "with `pip3 install twine`" as an easy installation instruction!